test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMessageWriterTests.cs

```diff
@@ -14,6 +14,10 @@
 using Microsoft.OData.Tests.Json;
 using System.Text;
 using Microsoft.Test.OData.DependencyInjection;
+using Microsoft.OData.UriParser;
+#if NETCOREAPP3_1_OR_GREATER
+using System.Text.Json;

```

Comment:
It could be sorting the usings, right?

Reply:
I'll sort the `using`s above the `#if`. Do you also suggesting sorting the `#if`? I think it makes sense to group the `#if` separately?